docs/sources/tempo/api_docs/_index.md

```diff
@@ -591,16 +593,19 @@ You can pass an optional URL query parameter, `q`, to your request.
 The `q` parameter is a URL-encoded [TraceQL query]({{< relref "../traceql" >}}).
 If provided, the tag values returned by the API are filtered to only return values seen on spans matching your filter parameters.
 
-Queries can be incomplete: for example, `{ .cluster = }`. Tempo extracts only the valid matchers and build a valid query.
+Queries can be incomplete: for example, `{ resource.cluster = }`.
+Tempo extracts only the valid matchers and builds a valid query.
```

Comment:
For me it'd be helpful to mention that it won't throw error in case of invalid formation.
In case of failure it'll show the whole list.

Reply:
```suggestion
Tempo extracts only the valid matchers and builds a valid query.
If an input is invalid, Tempo doesn't provide an error. Instead, 
you'll see the whole list when a failure of parsing input. 
```

Reply:
That's great

Reply:
Done. Thank you for the feedback.